Week1/main.py

```diff
@@ -0,0 +1,57 @@
+#1.print my name and age
+name = "Hema"
+age = 37
+print(f"Name:{}\n Age:{}".format(name, age))
+
+#2. Create a list of your 5 favorite movies and store it in the variable
+fav_list = ["Titanic", "Home alone", "Silence", "Dinosaur", "Frozen"]
+
+#3. Display the first and last colors from the  list.
+color_list = ["Red", "Green", "White" , "Black"]
+print(color_list[0])
+print(color_list[-1])
+
+#4. A Python script to add a key to a dictionary
+sample_dictionary= {0: 10, 1: 20}
+sample_dictionary[2] = 30
+print(sample_dictionary)
+
+#5. Python program to calculate body mass index.
+height = float(input("Enter your body weight in kg"))
+weight = float(input("Enter your body height in feet"))
+print("Your body mass index is: ", round(weight / (height * height)))
```

Comment:
Python has a method to write `height * height` more elegantly, try to use the power to operator